api/src/service/domain/permissions.ts

```diff
@@ -1,18 +1,22 @@
 import Joi = require("joi");
-
-import Intent, { allIntents } from "../../authz/intents";
+import Intent, { allIntents, ExposableIntent } from "../../authz/intents";
 import { Identity } from "./organization/identity";
 
 export type Permissions = { [key in Intent]?: Identity[] };
+export type ExposablePermissions = { [key in ExposableIntent]?: Identity[] };
 
 export const permissionsSchema = Joi.object().pattern(
   Joi.valid(allIntents),
   Joi.array().items(Joi.string()),
 );
 
-export const filterPermissions = (permissions: Permissions, filter: string[]): Permissions => {
+export const getExposablePermissions = (
+  permissions: Permissions,
+  filter: Array<"project.close" | "subproject.close" | "workflowitem.close">,
+): ExposablePermissions => {
+  const filterablePermissions = filter as string[];
```

Comment:
what is that, write a small function description about getExposablePermissions

why filterablePermissions? Edit the type of filter instead in Line 15

Are you sure